frontend/src/components/widgets/Multiselect/Multiselect.test.tsx

```diff
@@ -288,4 +288,116 @@ describe("Multiselect widget", () => {
       }
     )
   })
+  describe("properly invalidates going over max selections", () => {
+    it("has correct noResultsMsg when maxSelections is not passed", () => {
+      const props = getProps(
+        MultiSelectProto.create({
+          id: "1",
+          label: "Label",
+          default: [0],
+          options: ["a", "b", "c"],
+        })
+      )
+      const wrapper = mount(<Multiselect {...props} />)
+
+      expect(
+        wrapper
+          .find(UISelect)
+          .prop("noResultsMsg")
+          ?.toString()
+      ).toBe("No results")
+    })
+
+    it("has correct noResultsMsg when maxSelections is passed", () => {
+      const props = getProps(
+        MultiSelectProto.create({
+          id: "1",
+          label: "Label",
+          default: [0, 1],
+          options: ["a", "b", "c"],
+          maxSelections: 2,
+        })
+      )
+      const wrapper = mount(<Multiselect {...props} />)
+
+      expect(
+        wrapper
+          .find(UISelect)
+          .prop("noResultsMsg")
+          ?.toString()
+      ).toBe(`You can only select up to 2 options. Remove an option first.`)
```

Comment:
nit (here and below): No need to use a format string here since it's just a fixed result